.github/dependabot.yml

```
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "pulumi"
+    schedule:
+      interval: "weekly"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)